### ISSUE TITLE
sq.im.ImageContainer loads data incorrectly 

### ISSUE THREAD

**Issue by MxMstrmn** [theislab/squidpy#389]
### Description

The `sq.im.ImageContainer` loads high-res Visium images (`.tif`) incorrectly - images show an unexpected checkerboard pattern. This issue persists across different Visium datsets. Other image libraries (like `imageio`) show that the image file is not corrupted. The same image could be successfully loaded in previous versions of `squidpy`. 

### Minimal reproducible example

```python
import imageio
import os 
from os.path import join 
import matplotlib.pyplot as plt 
import scanpy as sc
import squidpy as sq

# Adjust paths according to your system 
storage = '/storage/groups/ml01/datasets/raw/20200909_PublicVisium_giovanni.palla'
folder = "2020_10XFluoVisium_HumanBrain1_giovanni.palla"
dataset_name = 'V1_Human_Brain_Section_1'


image_path = join(join(storage, folder), dataset_name+'_image.tif')
print(f"Valid image path: {os.path.exists(image_path)}")

image = sq.im.ImageContainer(image_path, lazy=True, chunks=200)

display(image)

plt.imshow(image['image'][5800:6000, 5800:6000, 0, 0])

img = imageio.volread(image_path)
plt.imshow(img[0, 5800:6000, 5800:6000])
```
Output from `ImageContainer`:
<img src="https://user-images.githubusercontent.com/24990533/127053255-1ad87349-2b7d-4234-9174-f0d2bbf2549a.png" data-canonical-src="https://gyazo.com/eb5c5741b6a9a16c692170a41a49c858.png" width="300" height="300" />
Expected output:
<img src="https://user-images.githubusercontent.com/24990533/127053360-79082c09-9786-49eb-a8c3-075f9c537aeb.png" data-canonical-src="https://gyazo.com/eb5c5741b6a9a16c692170a41a49c858.png" width="300" height="300" />
### Traceback

<details>

```pytb
no error thrown
```

</details>

### Version

1.1.0

**Comment by michalk8** [comment]
@MxMstrmn should be fixed in #390 (tried with your data, looks correct, still need to write tests though).

**Comment by michalk8** [comment]
closed via #390
